icloudpd/email_notifications.py

```diff
@@ -15,7 +15,7 @@ def send_2sa_notification(
     from_addr = smtp_email if smtp_email else to_addr
     logger = setup_logger()
     logger.info("Sending 'two-step expired' notification via email...")
-    smtp = smtplib.SMTP()
+    smtp = smtplib.SMTP(smtp_host, smtp_port)
     smtp.set_debuglevel(0)
     smtp.connect(smtp_host, smtp_port)
```

Comment:
smtp.connect is implicit now. Explicit call should be removed.

Reply:
The best thing to do is to make the changes yourself. Close here.